OData/src/System.Web.OData/OData/Query/Expressions/AggregationBinder.cs

```diff
@@ -253,6 +273,18 @@ private Expression CreateAggregationExpression(ParameterExpression accum, Aggreg
             return aggregationExpression;
         }
 
+        private Expression WrapDynamicCastIfNeeded(Expression propertyAccessor)
+        {
+            if (propertyAccessor.Type == typeof(object))
+            {
+                var convertToType = typeof(decimal?);
+
+                return Expression.Call(null, ExpressionHelperMethods.ConvertToDecimal, propertyAccessor);
```

Comment:
It uses it. ExpressionHelperMethods.ConvertToDecimal is MethodInfo for SafeConvertToDecimal. Need to keep name different, because we can't have method and property with the same name